openpype/hosts/nuke/plugins/publish/extract_slate_frame.py

```diff
@@ -98,18 +108,65 @@ def render_slate(self, instance):
         if "#" in fhead:
             fhead = fhead.replace("#", "")[:-1]
 
-        previous_node = node
+        # Create a text file in staging dir to use as a
```

Comment:
Few comments regarding this. There is no reason why to store clipboard path to instance.data. Also adding temp file to staging dir is not a good idea. Then it seems that the clipboard path is used for duplication of nodes, if it's so then there is a function `duplicate_node` in `openpype.hosts.nuke.api.lib`. If it's not enough then function `_duplicate_node_temp` can be made as public and used here. Which does actually similar thing.

Reply:
unfortunately i've seen that the code changes we did are built on 3.11.1, whereas develop changed a lot of the stuff in extract_slate_frame. There was no function like that at that time.
I need to test this on develop to actually see if this just works as we would like and propose the fixes there.